po/fr.po

```diff
@@ -344,7 +344,7 @@ msgid "Use theme colors for foreground/background"
 msgstr "Utiliser les couleurs du thème système"
 
 msgid "Strip first character of paste if comment or variable declaration"
-msgstr "Strip first character of paste if comment or variable declaration"
+msgstr "Supprime le premier caractère si commentaire ou déclaration de variable"
```

Comment:
Should be something like this `Supprimer le premier caractère si ceci un commentaire ou une déclaration de variable`